<ui version="4.0" >
 <class>wepConfig</class>
 <widget class="QDialog" name="wepConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WEP Configuration</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="wificonfig.qrc" >
    <normaloff>:/tray_wifi85.png</normaloff>:/tray_wifi85.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Wireless Network Key</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QRadioButton" name="radioHex" >
        <property name="text" >
         <string>Hex Key</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2" >
       <widget class="QRadioButton" name="radioPlaintext" >
        <property name="text" >
         <string>Plaintext</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Network Key</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3" >
       <widget class="QLineEdit" name="lineKey" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Network Key (Repeat)</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3" >
       <widget class="QLineEdit" name="lineKey2" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="textLabel1_2_2" >
        <property name="text" >
         <string>Key Index</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3" >
       <widget class="QSpinBox" name="spinIndex" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4" >
       <spacer name="spacer11" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>61</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="5" >
       <widget class="QCheckBox" name="checkShowKey" >
        <property name="text" >
         <string>Show Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="spacer12" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>161</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <includes>
  <include location="local" >qmessagebox.h</include>
  <include location="local" >QGroupBox</include>
 </includes>
 <resources>
  <include location="wificonfig.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>wepConfig</receiver>
   <slot>slotClose()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkShowKey</sender>
   <signal>clicked()</signal>
   <receiver>wepConfig</receiver>
   <slot>slotShowKey()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel" >
     <x>170</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
